app/services/xccdf_report_parser.rb

```diff
@@ -33,7 +33,7 @@ def test_result
     begin
       @test_result = ::OpenSCAP::Xccdf::TestResult.new(source)
     rescue ::OpenSCAP::OpenSCAPError => e
-      Sidekiq.logger.error('Error: ', e)
+      Sidekiq.logger.error("Error: #{e}")
```

Comment:
By pushing for more coverage, I've covered this error with tests and realized we were using this method incorrectly. Doh!